2-dom-manipulation/form.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const NAME_REGEX = /^(([A-Za-z\s])+)$/;
+const EMAIL_REGEX = /([a-z0-9]+[_az0-9\.-]*[a-z0-9]+)@([a-z0-9-]+(?:\.[a-z0-9-]+)*\.[a-z]{2,4})/;
+const PHONE_NUMBER_REGEX = /^([0-9]{10})$/;
+
+const HTML_FORM = document.querySelector('.form');
+HTML_FORM.addEventListener('submit', validateForm);
+
+const NAME_ERROR_CLASS = 'name-error';
+const EMAIL_ERROR_CLASS = 'email-error';
+const PHONE_ERROR_CLASS = 'phone-error';
+const COURSES_ERROR_CLASS = 'fieldset-courses-legend-error';
+const FIELD_COURSES_LEGEND = document.querySelector('.field-courses legend');
```

Comment:
```suggestion
const CLASSES = {
NAME_ERROR_CLASS : 'name-error',
EMAIL_ERROR_CLASS : 'email-error',
PHONE_ERROR_CLASS : 'phone-error',
COURSES_ERROR_CLASS : 'fieldset-courses-legend-error',
FIELD_COURSES_LEGEND : document.querySelector('.field-courses legend')
};
```
You can use enums or objects in JS and use `CLASSES. NAME_ERROR_CLASS` where you need the value.